showcase-v1alpha2/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "repository": "googleapis/nodejs-showcase",
+  "name": "showcase",
+  "version": "0.1.0",
+  "author": "Google LLC",
+  "description": "GAPIC Showcase API client for Node.js",
+  "main": "build/src/index.js",
+  "files": [
+    "protos",
+    "src",
```

Comment:
Do you really want `src`, or `build/src`?

Reply:
I have no idea! What does this `"main"` key do?

Reply:
`main` explains the entry point of the npm module.  That is correct.  The `files` group says what to include in the npm module.  For that, I am guessing you want `build/src` instead of `src`.